ext/pdo_mysql/get_error_codes.php

```diff
@@ -6,7 +6,7 @@
     while (!feof(STDIN)) {
         $line = fgets(STDIN);
 
-        if (ereg('^\{[[:space:]]+(ER_.*)[[:space:]]+,[[:space:]]*"(.*)",[[:space:]]*"(.*)"', $line, $matches)) {
+        if (preg_match('^\{[[:space:]]+(ER_.*)[[:space:]]+,[[:space:]]*"(.*)",[[:space:]]*"(.*)"', $line, $matches)) {
```

Comment:
I don't think this can work; at least `preg_match()` requires delimiters, while `ereg()` does not.

I'm not sure though, if keeping this file makes sense. @kamil-tekiela, thoughts?

Reply:
What brought you to this file? Why do you need to execute it?

Reply:
@kamil-tekiela I was just studying the extension and trying to understand it, I came across this function that was already removed in version 7.

If you find this change unnecessary, I don't see any problem.